flixel/FlxCamera.hx

```diff
@@ -1524,6 +1524,15 @@ class FlxCamera extends FlxBasic
 		updateFlashOffset();
 		setScale(scaleX, scaleY);
 	}
+
+	/**
+	 * Checks whether this camera contains a given point or rectangle, in
+	 * screen coordinates.
+	 */
+	public inline function containsPoint(point:FlxPoint, width:Float = 0, height:Float = 0):Bool
+	{
+		return (point.x + width >= 0) && (point.x < this.width) && (point.y + height >= 0) && (point.y < this.height);
```

Comment:
It seems like you also changed this logic to be inclusive? (`>= 0` instead of just `> 0`). This leads to unexpected behavior in `FlxObject#isOnScreen()`:

```haxe
var object = new FlxObject(0, 0, 1, 1);
add(object);
trace(object.isOnScreen()); // true
object.x = -1;
trace(object.isOnScreen()); // should be false, but true with >= 0
```